components/places/src/types.rs

```diff
@@ -44,6 +44,12 @@ where
     }
 }
 
+impl From<SyncGuid> for dogear::Guid {
+    fn from(guid: SyncGuid) -> dogear::Guid {
```

Comment:
I almost feel like we should just use dogear::Guid everywhere, although I don't actually that would work in practice.